staging/src/k8s.io/kube-apiextensions-server/test/integration/testserver/resources.go

```diff
@@ -28,19 +28,38 @@ import (
 	"k8s.io/kube-apiextensions-server/pkg/client/clientset/clientset"
 )
 
-func NewNoxuCustomResourceDefinition() *apiextensionsv1alpha1.CustomResource {
+func NewNoxuNamespacedCustomResourceDefinition() *apiextensionsv1alpha1.CustomResource {
 	return &apiextensionsv1alpha1.CustomResource{
 		ObjectMeta: metav1.ObjectMeta{Name: "noxus.mygroup.example.com"},
 		Spec: apiextensionsv1alpha1.CustomResourceSpec{
 			Group:   "mygroup.example.com",
 			Version: "v1alpha1",
+			Scope:   apiextensionsv1alpha1.NamespaceScoped,
 			Names: apiextensionsv1alpha1.CustomResourceNames{
-				Plural:   "noxus",
-				Singular: "nonenglishnoxu",
-				Kind:     "WishIHadChosenNoxu",
-				ListKind: "NoxuItemList",
+				Plural:     "noxus",
+				Singular:   "nonenglishnoxu",
+				Kind:       "WishIHadChosenNoxu",
+				ShortNames: []string{"shortnoxu", "shortnoxu2"},
+				ListKind:   "NoxuItemList",
+			},
+		},
+	}
+}
+
+func NewNoxuClusterCustomResourceDefinition() *apiextensionsv1alpha1.CustomResource {
```

Comment:
Also here I would merge these two funcs and add `scope` parameter to `NewNoxuCustomResourceDefinition`.